public/js/reducers/management.js

```diff
@@ -10,73 +10,59 @@ export const initialMgmtState = {
 
 export default function management(state, action) {
   state = state || initialMgmtState;
-  var newState;
 
   switch (action.type) {
     case actionTypes.USER_SIGNED_IN:
       if (!state.tab) {
-        newState = Object.assign({}, initialMgmtState, {
+        return Object.assign({}, initialMgmtState, {
           tab: 'profile',
           view: actionTypes.SHOW_MY_ACCOUNT,
         });
       }
       break;
```

Comment:
this technically is a reachable break because `state.tab` could be set. I thought it was a little nitpicky to remove all the unreachable breaks but eslint wouldn't pass without it :/